guides/index.md

```diff
@@ -7,9 +7,9 @@ sidebar_position: 1
 
 - [Sending Enrollment Emails](/guides/send-enrollment)
 - [Integrate Auth into a bubble.io application](/guides/bubble-io-integration.md)
+- [Integrate Auth into NextAuth](/guides/integrate-nextauth.md)
 - [Integrate Auth into Wordpress](/guides/integrate-wordpress.md)
```

Comment:
This is out of the scope of this PR, but I'm wondering if we should call this `"Integrate Beyond Identity into X..."` instead of just Auth? Since we have other SSO  integrations below it seems like we should make it clear this is a Beyond Identity integration. What do you think @nmelo @ismailmustafa @Will5 ?

<img width="544" alt="Screenshot 2023-01-05 at 11 35 27 AM" src="https://user-images.githubusercontent.com/11654201/210832274-dddca6f5-5b4e-4cb4-b8d2-64e90f2f28b8.png">

Reply:
Just noticed the side bars are supposed to say something like "Integrate Beyond Identity Passwordless Authentication into X..." but they aren't showing up on the side bar. Maybe we should make this the link title too?

Reply:
So we don't block this PR I created a new one to discuss this: https://github.com/gobeyondidentity/developer-docs/pull/200